entity-framework/core/managing-schemas/migrations/index.md

```diff
@@ -1,294 +1,184 @@
-
-## Create a migration
-
-After you've [defined your initial model](xref:core/modeling/index), it's time to create the database. To add an initial migration, run the following command.
-
-### [.NET Core CLI](#tab/dotnet-core-cli)
-
-```dotnetcli
-dotnet ef migrations add InitialCreate
-```
-
-### [Visual Studio](#tab/vs)
-
-``` powershell
-Add-Migration InitialCreate
+```c#
+public class Blog
+{
+    public int Id { get; set; }
+    public string Name { get; set; }
+}
 ```
 
-***
-
-Three files are added to your project under the **Migrations** directory:
+During development, you may have used the [Create and Drop APIs](xref:core/managing-schemas/ensure-created) to iterate quickly; but now that your application is going to production, you need a way to evolve the schema without dropping the entire database.
 
-* **XXXXXXXXXXXXXX_InitialCreate.cs**--The main migrations file. Contains the operations necessary to apply the migration (in `Up()`) and to revert it (in `Down()`).
-* **XXXXXXXXXXXXXX_InitialCreate.Designer.cs**--The migrations metadata file. Contains information used by EF.
-* **MyContextModelSnapshot.cs**--A snapshot of your current model. Used to determine what changed when adding the next migration.
+### Install the tools
 
-The timestamp in the filename helps keep them ordered chronologically so you can see the progression of changes.
+First, you'll have to install the [EF Core command-line tools](xref:core/miscellaneous/cli/index):
 
-### Namespaces
+* We generally recommend using the [.NET Core CLI tools](xref:core/miscellaneous/cli/dotnet), which work on all platforms.
+* If you're more comfortable working inside Visual Studio or have experience with EF6 migrations, you can also use the [Package Manager Console tools](xref:core/miscellaneous/cli/powershell).
 
-You are free to move Migrations files and change their namespace manually. New migrations are created as siblings of the last migration.
+### Create your first migration
 
-Alternatively you can use `-Namespace` (Package Manager Console) or `--namespace` (.NET Core CLI) to specify the namespace at generation time.
+You're now ready to add your first migration! Instruct EF Core to create a migrations named **InitialCreate**:
 
-### [.NET Core CLI](#tab/dotnet-core-cli)
+#### [.NET Core CLI](#tab/dotnet-core-cli)
 
 ```dotnetcli
-dotnet ef migrations add InitialCreate --namespace Your.Namespace
+dotnet ef migrations add InitialCreate
 ```
 
-### [Visual Studio](#tab/vs)
+#### [Visual Studio](#tab/vs)
 
 ``` powershell
-Add-Migration InitialCreate -Namespace Your.Namespace
+Add-Migration InitialCreate
 ```
 
 ***
 
-## Update the database
+EF Core will create a directory called **Migrations**, and generated some files. In general, it's a good idea to inspect what exactly EF Core generated - and possibly amend it - but we'll skip over that for now.
 
-Next, apply the migration to the database to create the schema.
+### Create a migration SQL script
```

Comment:
Note that the tutorial shows the SQL script flow, rather than the possibly "simpler" ones. I hope this pushes new users to work in the recommended way right away.

Reply:
Am modifying the tutorial to do this.